service-front/app/features/bootstrap/behat.config.php

```diff
@@ -29,10 +29,11 @@
     ],
 
     'feature_flags' => [
-        'use_older_lpa_journey' => true,
-        'delete_lpa_feature'    => true,
-        'allow_older_lpas'      => true,
-        'save_older_lpa_requests' => true,
+        'use_older_lpa_journey'             => true,
+        'delete_lpa_feature'                => true,
+        'allow_older_lpas'                  => true,
+        'save_older_lpa_requests'           => true,
+        'streamline_cleansing_lpas'         => true,
```

Comment:
```suggestion
        'use_older_lpa_journey'     => true,
        'delete_lpa_feature'        => true,
        'allow_older_lpas'          => true,
        'save_older_lpa_requests'   => true,
        'streamline_cleansing_lpas' => true,
```
Should only add as many extra spaces as are needed for the longest line to make sense.